utils/aeromatic++/Aircraft.cpp

```diff
@@ -249,6 +249,21 @@ bool Aeromatic::fdm()
 
 
 //***** METRICS ***************************************
+    // empty weight > max weight? then assume a user mistake.
+    if (_empty_weight > _max_weight)
+    {
+        float tmp = _max_weight;
+        _max_weight = _empty_weight;
+        _empty_weight = tmp;
+        _warnings.push_back("Empty weight is set larger than maximum weigth, swapping.");
+    }
+
+    if (_max_weight == 0)
+    {
+        _alerts.push_back("Maximum weigth is set to zero. Guessing.");
+        _max_weight = 10000.0f;
+    }
```

Comment:
I'd move this block before the test `if (_empty_weight > _max_weight)` above because if one defines an empty weigth of 11000 with no max weight then they would end up with an empty weight of 11000 and a max weight of 10000 and this would trigger the bug reported in #787

Reply:
With that input combination empty weight and max weight would get swapped making empty weight == 0 and max weight == 11000 which triggers the test further down the code:
```
    // estimate empty weight, based on max weight
    if (_empty_weight == 0) {
        _empty_weight = _max_weight * aircraft->get_empty_weight();
    }
```

Testing it for real results in:
```
    Maximum takeoff weight             : 11000.00
    Empty weight                       : 9240.00
```

Reply:
Indeed. Thanks for your answer.